aptos-move/aptos-vm/src/aptos_vm_impl.rs

```diff
@@ -285,7 +284,8 @@ impl AptosVMImpl {
             .execute_function_bypass_visibility(
                 &chain_specific_info.module_id(),
                 prologue_function_name,
-                gas_currency,
+                // TODO: Deprecate this once we remove gas currency on the Move side.
+                vec![],
```

Comment:
Have we made the move change yet?

Reply:
Nope that's why I'm keeping vec![] here. I'll look into the change on Move side separately